doc/fuzzing.md

```diff
@@ -28,9 +32,13 @@ export AFL_HARDEN=1
 cd src/
 make test/test_bitcoin_fuzzy
 ```
-We disable ccache because we don't want to pollute the ccache with instrumented
-objects, and similarly don't want to use non-instrumented cached objects linked
-in.
+
+Disabling ccache should be optional here, and is added above solely to save some HDD
+space.
+
+The fuzzer is _a lot_ faster now when run in LLVM persistent mode. To enable LLVM persistent mode,
+bitcoin_test_fuzzy has to be built with clang/clang++. For this replace the
+above mentions of `afl-gcc` and `afl-g++` with `afl-clang-fast` and afl-clang-fast++`.
```

Comment:
`afl-clang-fast` and `afl-clang-fast++` are not generated in the compilation process described at line 30.

Reply:
just discovered that to get the `fast` version you need to get into the `llvm_mode` subdir inside the afl source tree

Reply:
Using the `afl` binaries ( compiled from source I got a little bit further but I got another error:

Reply:
Yes. In my PR, I didn't describe how to build `afl` properly. I think that's out of the scope of this PR?

To be honest, I also had some trouble here and there with triggering bugs in old versions of `gnu-as` and the like. So I wonder whether I should put a couple hints into the docs - and what I should hint at?

Maybe like - from our chat now - "you might want to try adding `-D__STRICT_ANSI__` when compiling with AFL based on `clang-3.8`?

For fast fuzzing, make sure you build AFL with `afl-clang-fast` enabled by building the `llvm_mode` subdirectory?

Reply:
1 quest) I agree, probably out of scope